.kokoro/osx.sh

```diff
@@ -28,6 +28,7 @@ function set_failed_status {
 }
 
 # Install/Use 2.5.5
+source /Users/kbuilder/.rvm/scripts/rvm
```

Comment:
Does this mean we can also install and build against 2.3, 2.4, and 2.6?

Reply:
It'll take forever, and my understanding is that the mac resources are much more limited than the GCP images. What if we tested all versions for nightly?